petastorm/spark/spark_dataset_converter.py

```diff
@@ -119,19 +119,23 @@ def make_tf_dataset(self,
             preproc_parallelism=preproc_parallelism
         )
 
-    def make_torch_dataloader(self):
+    def make_torch_dataloader(self, **petastorm_kwargs):
         """
         Make a PyTorch DataLoader.
 
         This method will do the following two steps:
           1) Open a petastorm reader on the materialized dataset dir.
           2) Create a PyTorch DataLoader based on the reader created in (1)
 
+        :param petastorm_kwargs: all the arguments for `make_batch_reader`
```

Comment:
How do we know how to split the kwargs?

Reply:
L213-218: DataLoader only has 3 keywords, so we will use it as a whitelist. Then we assume all the other keywords are for the reader.

Reply:
~~~python
make_torch_loader(*,
  batch_size=32, num_epochs=None, cur_shard=None, shard_count=None,
  preprocess_fn=None, **petastorm_reader_kwargs)
~~~